dotnet/console/Console/Scenarios/DownloadDocumentVersionScenario.cs

```diff
@@ -0,0 +1,53 @@
+﻿using Lacuna.Signer.Api;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Console.Scenarios
+{
+    class DownloadDocumentVersionScenario : Scenario
+    {
+        /**
+         * This scenario demonstrates how to download different document versions 
+         */
+
+        public override async Task RunAsync()
+        {
+            // 1. Get a document Id
+            var result = await CreateDocumentAsync();
+
+            // 2. You can directly get an  especific document ticket 
```

Comment:
@jrodriguesmagal podia explicar o que é o ticket:
You can get a ticket to a specific version of the document. The ticket is a temporary URL that allows you to download that version.